TGM/src/main/java/network/warzone/tgm/modules/killstreak/KillstreakModule.java

```diff
@@ -98,7 +98,7 @@ public void onKill(TGMPlayerDeathEvent event) {
 
     public int getKillstreak(String uuid) {
         if (!players.containsKey(uuid) || players.get(uuid) == null) {
-            return 0;
+            return -1;
```

Comment:
What is the advantage of returning a -1 instead of a 0? Could also use `getOrDefault()`

Reply:
I wanted to return either -1, 0 or >0 for some reason, but now I guess there was no use for it. I'll just change it and add the getOrDefault()